src/api/undeploy/helpers/undeploy-service-from-environment.test.js

```diff
@@ -0,0 +1,96 @@
+import { registerUndeployment } from '~/src/api/undeploy/helpers/register-undeployment.js'
+import { removeDeploymentFile } from '~/src/api/undeploy/helpers/remove-deployment-file.js'
+import { lookupTenantService } from '~/src/api/deploy/helpers/lookup-tenant-service.js'
+import { isFeatureEnabled } from '~/src/helpers/feature-toggle/is-feature-enabled.js'
+import { undeployServiceFromEnvironmentWithId } from '~/src/api/undeploy/helpers/undeploy-service-from-environment.js'
+
+const logger = {
+  info: jest.fn(),
+  error: jest.fn()
+}
+
+jest.mock('~/src/helpers/feature-toggle/is-feature-enabled', () => {
+  return {
+    isFeatureEnabled: jest.fn()
+  }
+})
+jest.mock('~/src/api/undeploy/helpers/register-undeployment', () => {
+  return {
+    registerUndeployment: jest.fn()
+  }
+})
+jest.mock('~/src/api/undeploy/helpers/remove-deployment-file', () => {
```

Comment:
I'm pretty sure all of these can be `jest.mock('~/src/api/undeploy/helpers/remove-deployment-file')` which mocks all items in the file

Reply:
:confused:

Reply:
I am not sure I get this one